app/routes/image-set.js

```diff
@@ -3,6 +3,17 @@ import OrganizationRouteMixin from 'lion-guardians/mixins/organization-route';
 import RequireUserMixin from 'lion-guardians/mixins/require-user';
 
 export default Ember.Route.extend( OrganizationRouteMixin, RequireUserMixin, {
+  beforeModel: function() {
+    var route = this;
+    return this._super().then(function() {
+      if (window.SKIP_MAPS) {
+        return Ember.RSVP.resolve();
+      } else {
+        return route.loadGoogleMap();
```

Comment:
@bantic  This lazy loads the Google Map only when we enter an image-set route. However I want to push this down further and only load when a user clicks the edit button on an image-set. The plugin only injects the loadGoogleMap() function on routes, so I was wondering the best way to do this. I would either inject it on components so it can be called in the lg-google-map component, or make another action on the controller and pass this down to the component. Is it possible to pass route actions to a component or only controller actions? Anyway let me know if a) you think this is necessary and b) how you would do it.